lib/file/widgets/file_tile.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:schulcloud/app/app.dart';
+
+import '../data.dart';
+import 'file_thumbnail.dart';
+
+class FileTile extends StatelessWidget {
+  const FileTile({Key key, @required this.file, @required this.onOpen})
+      : assert(file != null),
+        assert(onOpen != null),
+        super(key: key);
+
+  final File file;
+  final void Function(File file) onOpen;
+
+  void _showDetails(BuildContext context) {
+    showModalBottomSheet(
+      context: context,
+      useRootNavigator: true,
+      builder: (context) => Column(
+        mainAxisSize: MainAxisSize.min,
+        children: <Widget>[
+          SizedBox(height: 16),
+          ListTile(
+            title: Text(file.name),
+            subtitle: Text(
+              '${file.isDirectory ? '' : '${file.sizeAsString}\n'}'
+              'created at ${dateTimeToString(file.createdAt)}\n'
+              'last modified at ${dateTimeToString(file.updatedAt)}',
```

Comment:
These should also display the time

Reply:
We'll probably do this at a later time. The `timeago` package could also be useful to automatically generate strings like `yesterday at xx clock`.